explorer/websocket.go

```diff
@@ -294,23 +301,37 @@ func (wsh *WebsocketHub) run() {
 				wsh.bufferMtx.Unlock()
 				continue
 			}
+
+			// Copy the transaction slice and make a new empty buffer.
 			txs := make([]*types.MempoolTx, len(wsh.newTxBuffer))
 			copy(txs, wsh.newTxBuffer)
```

Comment:
This copy is actually unnecessary since `wsh.newTxBuffer` gets overwritten with a new slice below.  Here it could be `txs := wsh.newTxBuffer`, but I don't want to fiddle with this now.